singularity-opac/neutrinos/opac_neutrinos.hpp

```diff
@@ -16,7 +16,7 @@
 #ifndef SINGULARITY_OPAC_NEUTRINOS_OPAC_NEUTRINOS_
 #define SINGULARITY_OPAC_NEUTRINOS_OPAC_NEUTRINOS_
 
-#include <variant/include/mpark/variant.hpp>
+#include "../../utils/variant/include/mpark/variant.hpp"
```

Comment:
Should this relative path be changed? I think the global path should still work?

Reply:
Ah you already fixed it. :+1:

Reply:
already fixed :)